cognite_toolkit/_cdf_tk/commands/auth.py

```diff
@@ -133,20 +133,25 @@ def check_auth(
 
         print(f"[italic]Focusing on current project {auth_vars.project} only from here on.[/]")
         print(
-            "Checking basic project and group manipulation access rights (projectsAcl: LIST, READ and groupsAcl: LIST, READ, CREATE, UPDATE, DELETE)..."
+            "Checking basic project and group manipulation access rights "
+            "(projectsAcl: LIST, READ and groupsAcl: LIST, READ, CREATE, UPDATE, DELETE)..."
         )
         try:
             ToolGlobals.verify_client(
                 capabilities={
-                    "projectsAcl": ["LIST", "READ"],
-                    "groupsAcl": ["LIST", "READ"],
+                    "projectsAcl": [
+                        "LIST",
+                        "READ",
+                    ],
+                    "groupsAcl": ["LIST", "READ", "UPDATE", "DELETE"],
```

Comment:
This is an extra bug I found. In the text it says it is checking list, read, create, update, delete, but it was only checking list and read.